_data/locales/en.yml

```diff
@@ -16,7 +16,7 @@ en:
     caskcreate: Making a cask is as simple as creating a formula.
     install:
       install: Install Homebrew
-      paste: Paste that in a macOS Terminal prompt.
+      paste: Paste that in a macOS Terminal or Linux shell prompt.
       what: The script explains what it will do and then pauses before it does it. Read about other <a href='https://docs.brew.sh/Installation'>installation options</a>. Install Homebrew on <a href='https://docs.brew.sh/Homebrew-on-Linux'>Linux and Windows Subsystem for Linux</a>.
```

Comment:
```suggestion
      what: The script explains what it will do and then pauses before it does it. Read about other <a href='https://docs.brew.sh/Installation'>installation options</a>.
```